netlify.toml

```diff
@@ -1,16 +1,13 @@
-[build.environment]
-  NODE_VERSION = "18"
-
 [build]
   publish = "./dist"
-  command = "npx pnpm i --frozen-lockfile --store=node_modules/.pnpm-store && npx pnpm run build"
+  command = "npm run build"
+  [build.environment]
+    NODE_VERSION = "18"
 
 [dev]
   publish = "dist/"
-  functions = "functions"
   targetPort = 3000
 
 [functions]
   directory = "functions"
   node_bundler = "esbuild"
-  external_node_modules = ["@octokit/core", "isomorphic-fetch"]
```

Comment:
@Hrishikesh-K we should not expose anymore the dependencies ?

Reply:
Based on this: https://docs.netlify.com/configure-builds/file-based-configuration/#functions,

`external_node_modules` sets a list of Node.js modules that are copied to the bundled artifact without adjusting their source or references during the bundling process; only applies when `node_bundler` is set to `esbuild`. This property helps handle dependencies that can’t be inlined, such as modules with native add-ons.

Since your dependencies were being normally bundled, I'm not sure why you had this in the first place.